kubernetes/linux/Dockerfile.multiarch

```diff
@@ -88,26 +85,25 @@ COPY --from=builder /lib/libreadline.so.8 /lib/
 COPY --from=builder /usr/lib/libncursesw.so.6 /usr/lib/libtinfo.so.6 /usr/lib/
 # inotifywait dependencies
 COPY --from=builder /lib/libinotifytools.so.0 /lib/
+COPY --from=builder /lib/libc.so.6 /lib/

```

Comment:
is this new dependency for inotifywait?  and how did you detect this new dependency?

Reply:
It is not a new dependency. While deleting the vim package dependency, I just wanted to make sure that we add all the dependencies for all packages. 

For example, libs.so.6 is a dependency for many packages. I missed copying it multiple times before since it was present for some package. But, I realised now that we might want to remove complete packages so that shouldn't affect other package dependencies. Just ran this: https://github.com/microsoft/Docker-Provider/blob/ci_prod/MARINER.md#get-dependencies-for-the-distroless-image